presentation/feedprofile/src/main/java/com/hilingual/presentation/feedprofile/profile/FeedProfileScreen.kt

```diff
@@ -218,9 +270,25 @@ private fun FeedProfileScreen(
                                 state = pagerState,
                                 modifier = Modifier.fillParentMaxSize()
                             ) { page ->
-                                val (diaries, emptyCardType, tabType) = when (page) {
-                                    0 -> Triple(uiState.sharedDiaries, FeedEmptyCardType.NOT_SHARED, DiaryTabType.SHARED)
-                                    else -> Triple(uiState.likedDiaries, FeedEmptyCardType.NOT_LIKED, DiaryTabType.LIKED)
+                                val diaries: ImmutableList<FeedDiaryUIModel>
+                                val emptyCardType: FeedEmptyCardType
+                                val tabType: DiaryTabType
+                                val listState: LazyListState
+
+                                when (page) {
+                                    0 -> {
+                                        diaries = uiState.sharedDiaries
+                                        emptyCardType = FeedEmptyCardType.NOT_SHARED
+                                        tabType = DiaryTabType.SHARED
+                                        listState = sharedDiaryListState
+                                    }
+
+                                    else -> {
+                                        diaries = uiState.likedDiaries
+                                        emptyCardType = FeedEmptyCardType.NOT_LIKED
+                                        tabType = DiaryTabType.LIKED
+                                        listState = likedDiaryListState
+                                    }
```

Comment:
```suggestion
                                val tabType = DiaryTabType.entries[page]
                                val listState: LazyListState

                                when (tabType) {
                                    DiaryTabType.SHARED -> {
                                        diaries = uiState.sharedDiaries
                                        emptyCardType = FeedEmptyCardType.NOT_SHARED
                                        listState = sharedDiaryListState
                                    }

                                    DiaryTabType.LIKED -> {
                                        diaries = uiState.likedDiaries
                                        emptyCardType = FeedEmptyCardType.NOT_LIKED
                                        listState = likedDiaryListState
                                    }
                                }
```
이렇게 하는게 좀 더 가독성 좋은 것 같은데 어때요~~??